openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+                    "duplicateAttribute": "重複する属性です。",
+                    "unableToPersistPolicy": "ポリシーをデータストアに永続化することができません。",
+                    "policiesUploaded": "ポリシーを正常にアップロードしました。",
+                    "policiesUploadFailed": "ポリシーのアップロードに失敗しました。",
+                    "importFailed": "インポートに失敗しました: __applicationName__ レルム内にポリシーセットが見つかりません。",
+                    "error": {
+                        "cantStartWithHash": "LDAP エンティティは # で始めることはできません。",
+                        "noName": "名前は必須です。",
+                        "noId": "IDは必須です。",
+                        "noResourceTypes": "1つ以上のリソースタイプを選択してください。",
+                        "noResources": "1つ以上のリソースを選択してください。",
+                        "noActions": "1つ以上のアクションを追加してください。",
+                        "noPatterns": "1つ以上のパターンを追加してください。"
+                    }
+                }
+            }
+        },
+        "AppConfiguration" : {
+            "Navigation" : {
+                "links" : {
+                    "commonTasks": "共通タスク",
+                    "realms": {
+                        "title": "レルム",
+                        "showAll": "すべて表示",
+                        "newRealm": "新規レルム",
+                        "viewAll": "すべて表示..."
+                    },
+                    "configure": {
+                        "title": "設定",
+                        "authentication": "認証",
+                        "server-defaults": "デフォルトサーバー",
+                        "global-services": "グローバルサービス"
+                    },
+                    "deployment": {
+                        "title" : "デプロイメント",
+                        "servers" : "サーバー",
+                        "sites" : "サイト"
+                    },
+                    "federation": "連携",
+                    "configuration": "設定",
+                    "sessions": "セッション",
+                    "dashboard" : "ダッシュボード",
+                    "users" : "ユーザー",
+                    "apps" : "アプリケーション",
+                    "allApps" : "全てのアプリケーション",
+                    "addMore" : "アプリケーションの追加",
+                    "groups" : "グループ",
+                    "tasksMenu" : "タスク",
+                    "allTasks" : "グループのキューにあるタスク",
+                    "myTasks" : "マイタスク",
+                    "startProcess" : "処理を開始",
+                    "oauthtokens" : "OAuth 2 トークンマネージャー",
+                    "uma" : "共有",
+                    "umaLinks": {
+                        "history" : "履歴",
+                        "resources" : "リソース",
+                        "requests" : "リクエスト"
+                    },
+                    "console": "コンソール"
+                },
+                "username" : {
+                    "label" : "サインイン"
+                }
+            }
+        }
+    },
+    "templates" : {
+        "user" : {
+            "LoginDialog" : {
+                "refreshOnLogin" : "ログイン時にリフレッシュする必要があります" 
+            },
+            "LoginTemplate" : {
+                "logintextconter" : "",
```

What is `logintextconter`? It is not in the original file.

---

The line is a typo, so delete that line.
```suggestion
            "LoginTemplate" : {
                "loginRemember" : "ユーザー名を記憶する。",
```